src/status_im/chat/events/input.cljs

```diff
@@ -236,77 +236,50 @@
                                                 (min (count input-text)))]
               (merge fx (update-text-selection new-db new-selection)))))))
 
-(defn- request-command-data
-  "Requests command data from jail"
-  [{:keys [bot-db] :contacts/keys [contacts] :as db}
-   {{:keys [command
-            metadata
-            args]
-     :as   content} :content
-    :keys  [chat-id group-id jail-id data-type event-after-creator message-id current-time]}]
-  (let [{:keys [dapp? dapp-url name]} (get contacts chat-id)
-        metadata        (merge metadata
-                               (when dapp?
-                                 {:url  (i18n/get-contact-translated chat-id :dapp-url dapp-url)
-                                  :name (i18n/get-contact-translated chat-id :name name)}))
-        owner-id        (:owner-id command)
-        bot-db          (get bot-db owner-id)
-        params          (merge (input-model/args->params content)
-                               {:bot-db   bot-db
-                                :metadata metadata})
-
-        command-message {:command    command
-                         :params     params
-                         :to-message (:to-message-id metadata)
-                         :created-at current-time
-                         :id         message-id
-                         :chat-id    chat-id
-                         :jail-id    (or owner-id jail-id)}
-
-        request-data    {:message-id   message-id
-                         :chat-id      chat-id
-                         :group-id     group-id
-                         :jail-id      (or owner-id jail-id)
-                         :content      {:command       (:name command)
-                                        :scope-bitmask (:scope-bitmask command)
-                                        :params        params
-                                        :type          (:type command)}}]
-    (commands-events/request-command-message-data db request-data
-                                                  {:data-type             data-type
-                                                   :proceed-event-creator (partial event-after-creator
-                                                                                   command-message)})))
-
-;; TODO (janherich) request-command-data functions and event need to be refactored, they are needlessly complicated
+;; function creating "message shaped" data from command, because that's what `request-command-message-data` expects
```

Comment:
Should we move this comment to function's doc?

Reply:
We can, but I eventually want this function to go away and create "normal" messages and command messages in a same way and shape from the beginning (it would require too much changes downstream in the message model, so I gave up on that for now).

There is a lot of code just mindlessly juggling keys from one map to another for the command message flow, with no real justification.